documentation/puppet_server_metrics_performance.markdown

```diff
@@ -38,13 +38,17 @@ If wait time increases but borrow time stays the same, your Server infrastructur
 
 If both wait and borrow times are increasing, something else on your server is causing requests to take longer to process. The longer borrow times suggest that Puppet Server is struggling more than before to process requests, which has a cascading effect on wait times. Correlate borrow time increases with other events whenever possible to isolate what activities might cause them, such as a Puppet code change.
 
-If you are setting up Puppet Server for the first time, start by increasing your Server infrastructure's capacity through additional JRubies (if your server has spare CPU and memory resources) or compile masters until your free JRubies are at least 0 and average number of free JRubies are at least 1. Once your system can handle its request volume, you can start looking into more specific performance improvements.
+If you are setting up Puppet Server for the first time, start by increasing your Server infrastructure's capacity through additional JRubies (if your server has spare CPU and memory resources) or compile masters until you have more than 0 free JRubies, and your average number of free JRubies are at least 1. Once your system can handle its request volume, you can start looking into more specific performance improvements.
 
 #### Adding more JRubies
 
-If you need to add JRubies, remember that Puppet Server is tuned by default to use one fewer than your total number of CPUs, with a maximum of 4 CPUs, for the number of available JRubies. You can change this by setting `max-active-instances` in [`puppetserver.conf`][], under the `jruby-puppet` section. Don' set this value to something greater than the number of CPUs, because if there aren't enough processes available, it won't matter how many JRubies in the pool there are — the server won't be able to use them.
+If you must add JRubies, remember that Puppet Server is tuned by default to use one fewer than your total number of CPUs, with a maximum of 4 CPUs, for the number of available JRubies. You can change this by setting `max-active-instances` in [`puppetserver.conf`][puppetserver.conf], under the `jruby-puppet` section.
```

Comment:
What does `with a maximum of 4 CPUs` mean? Does that mean the JRubies are capped at 3? This seemed to make sense to me yesterday - so I must not have had enough coffee today - but I just can't parse that clause.